prql-compiler/src/tests/test.rs

```diff
@@ -2166,7 +2166,20 @@ fn test_table_alias() {
       LEFT JOIN salaries ON salaries.emp_no = e.emp_no
     GROUP BY
       e.emp_no
-    "###)
+    "###);
+
+    assert_display_snapshot!((compile(r###"
+    from e=employees
```

Comment:
```suggestion
    from e = employees
```

Reply:
I've been doing it the `e=employees` way! I'm fine standardizing on either. 

I think the reason to pick the `e=employees` approach is that we don't have commas between each item, and having items contiguous helps groups them.

(I'll merge this now but totally open to doing whatever the consensus is)